Header header
uint8 FL
uint8 FR
uint8 BL
uint8 BR

bool FL_DIR
bool FR_DIR
bool BL_DIR
bool BR_DIR
